cfy_manager/components/rabbitmq/rabbitmq.py

```diff
@@ -64,6 +67,12 @@ def _init_service(self):
         remove_file(rabbit_config_path)
         systemd.systemctl('daemon-reload')
 
+        if not self._installing_manager():
```

Comment:
There is a generic function in validations.py you can take advantage of, take a look here:
https://github.com/cloudify-cosmo/cloudify-manager-install/blob/6038bb56706d60ba2e872da7ea9b23a7557acf02/cfy_manager/components/validations.py#L372

Reply:
I think that actually makes the code less readable, as it would have to be called with:
if _services_coexistence_assertion(MANAGER_SERVICE, ''):

I think it reads much more clearly as:
if self._installing_manager():

(or self._install_manager() if we want to change that)

Reply:
What I meant was having:
if _service_coexistence_assertion(RABBITMQ_SERVICE, MANAGER_SERVICE):
    ....
That way you specify the actual services you don't want to coexist on the same server, plus you won't have repeatable code

Reply:
From discussion in slack, will look into what to do with this on the next stage (multiple external rabbit), as at the moment we were both agreeing the naming could use some work.